src/main/java/org/osc/controller/nuage/api/NuageSecurityControllerApi.java

```diff
@@ -348,19 +370,17 @@ public InspectionPortElement getRedirectionTarget(InspectionPortElement inspecti
         return null;
     }
 
-    private boolean isRedirectionTargetRegistered(String inspectionPortOSId, Domain selectDomain)
+    private RedirectionTarget getRedirectionTarget(String inspectionPortOSId, Domain selectDomain)

```

Comment:
Script constructs such as _like_ and '%s' should not really be hardcoded like that.

Reply:
There is likely a better way to perform these queries. I will keep it this way for consistency with the remaining code, once we address it can be addresses everywhere.